payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayRepository.kt

```diff
@@ -66,32 +62,35 @@ internal class DefaultGooglePayRepository(
      * See [Google Pay API docs](https://developers.google.com/android/reference/com/google/android/gms/wallet/PaymentsClient#isReadyToPay(com.google.android.gms.wallet.IsReadyToPayRequest))
      * for more details.
      */
-    override fun isReady(): Flow<Boolean> {
-        val isReadyState = MutableStateFlow<Boolean?>(null)
+    override fun isReady(): Flow<Boolean> = flow {
+        emit(isReadyAsync())
+    }
```

Comment:
Used a flow builder here rather than a `MutableStateFlow`. This will cause the flow to immediately close after emitting its only value rather than stay open as if expecting other values.

We should replace this in a future breaking update with `isReadyAsync`.